test/handlers/car.spec.js

```diff
@@ -30,4 +30,108 @@ describe('CAR handler', () => {
+    const branchBlock1 = await encode({ value: { leaf: leafBlock1.cid }, codec: cbor, hasher })
+    const rootBlock = await encode({ value: [branchBlock0.cid, branchBlock1.cid], codec: cbor, hasher })
+    const blockstore = mockBlockstore([leafBlock0, leafBlock1, branchBlock0, branchBlock1, rootBlock])
+    const dagula = new Dagula(blockstore)
+    const url = new URL(`http://localhost/ipfs/${rootBlock.cid}`)
+    const path = url.pathname.replace(`/ipfs/${rootBlock.cid}`, '')
+    const ctx = { waitUntil, dagula, dataCid: rootBlock.cid, path, searchParams: url.searchParams }
+    const env = { DEBUG: 'true' }
+    const req = new Request(url, {
+      headers: {
+        Accept: 'application/vnd.ipld.car; order=dfs'
+      }
+    })
+    const res = await handleCar(req, env, ctx)
+    const reader = await CarReader.fromBytes(new Uint8Array(await res.arrayBuffer()))
+    const roots = await reader.getRoots()
+    assert.strictEqual(roots[0].toString(), rootBlock.cid.toString())
+
+    const blocks = []
+    for await (const b of reader.blocks()) {
+      blocks.push(b)
+    }
+    assert.strictEqual(blocks[0].cid.toString(), rootBlock.cid.toString())
+    assert.strictEqual(blocks[1].cid.toString(), branchBlock0.cid.toString())
+    assert.strictEqual(blocks[2].cid.toString(), leafBlock0.cid.toString())
+    assert.strictEqual(blocks[3].cid.toString(), branchBlock1.cid.toString())
+    assert.strictEqual(blocks[4].cid.toString(), leafBlock1.cid.toString())
+  })
+
+  // unk = unknown, so could change...we're just asserting here that we get a
+  // different ordering from dfs, which tells us that our accept param is being
+  // successfully passed to the library.
+  //
+  // Note: unk is technically also dfs, so this test could become invalid in
+  // the future.
+  it('serves a CAR with accept parameter order=unk', async () => {
+    const waitUntil = mockWaitUntil()
+
+    const leafBlock0 = await encode({ value: fromString('test0'), codec: raw, hasher })
+    const leafBlock1 = await encode({ value: fromString('test1'), codec: raw, hasher })
+    const branchBlock0 = await encode({ value: { leaf: leafBlock0.cid }, codec: cbor, hasher })
+    const branchBlock1 = await encode({ value: { leaf: leafBlock1.cid }, codec: cbor, hasher })
+    const rootBlock = await encode({ value: [branchBlock0.cid, branchBlock1.cid], codec: cbor, hasher })
+    const blockstore = mockBlockstore([leafBlock0, leafBlock1, branchBlock0, branchBlock1, rootBlock])
+    const dagula = new Dagula(blockstore)
+    const url = new URL(`http://localhost/ipfs/${rootBlock.cid}`)
+    const path = url.pathname.replace(`/ipfs/${rootBlock.cid}`, '')
+    const ctx = { waitUntil, dagula, dataCid: rootBlock.cid, path, searchParams: url.searchParams }
+    const env = { DEBUG: 'true' }
+    const req = new Request(url, {
+      headers: {
+        Accept: 'application/vnd.ipld.car; order=unk'
+      }
+    })
+    const res = await handleCar(req, env, ctx)
```

Comment:
minor: could verify the fancty content-type header is as expected here on the response